_posts/2022-11-09-OpenSearch-Partner-Highlight-Using-Cloaked-Search.md

```diff
@@ -0,0 +1,107 @@
+encryption (ALE) because it happens at the layer above the data store instead of below it.
+
+By separating the data and keys, companies can achieve meaningful defense-in-depth where the everyday problems
+of cloud misconfigurations, application vulnerabilities, and network breaches don't necessarily result in a
+data breach. This makes gaining access to the needed keys an entirely different and more difficult problem for
+an attacker.  
+
+If this approach were used more often, we wouldn't constantly have stories about attackers that get into a
+corporate network, move laterally within the network to a database server, and then scrape all of the data
+down for sale on the darknet. The companies who are implementing this correctly are the ones who aren't in
+the news.
+
+## Why Encrypt Data in OpenSearch?
+
+Search services are often an underprotected backdoor way to gain access to sensitive data. They frequently hold
+personal information like names and addresses. They can hold log data that includes web form submissions. They
+may hold communications records like emails, customer messages, and chats. They may also hold insider-trading
+fodder like roadmap info, financial projections, legal documents, and much more.
+
+A search service is akin to an attractive nuisance. Just as a swimming pool without a fence will attract
+neighborhood kids, search data without encryption will attract hackers and employees.
+
+The employee side is what we call the “curious insider” problem where trusted users ignore policy and bypass or
+abuse application-level access controls to peek at data they shouldn't. Database administrators, ops, customer
+support, engineering – in most companies, many of these roles can sneak peeks at sensitive data. And these
+trusted folks who scratch the itch of curiosity can get a company in just as much trouble as a hacker.
+
+![Cloaked Search Proxy]({{ site.baseurl }}/assets/media/blog-images/2022-11-09-OpenSearch-Partner-Highlight-Using-Cloaked-Search/cloaked-search-proxy.png){:class="img-centered"}
+
+Access to sensitive data should always be controlled. Backdoor access should be eliminated whenever possible.
+That includes controlling access by admins. With application-layer encryption, a company can build in these
+controls for all users and administrators, and can fortify legitimate access with unbypassable audit trails.
+
+## Protecting Search Data with Cloaked Search
+
+At IronCore Labs, we provide tools and technologies that help companies protect the data they hold with
+application-layer encryption while keeping that data usable. One way we do that is to enable search over
+encrypted data.
+
+[Cloaked Search](https://ironcorelabs.com/products/cloaked-search/) by IronCore Labs is a drop-in proxy that
+sits in front of OpenSearch and automatically encrypts the sensitive data before it gets to the OpenSearch
+service. It's application-layer encryption for search. To search the data then requires a valid key. If you
```

Comment:
Instead of "To search the data then requires a valid key", I suggest "A valid key is then required in order to search the data."